sensor/common/sensor/sensor.go

```diff
@@ -432,8 +458,9 @@ func (s *Sensor) communicationWithCentralWithRetries(centralReachable *concurren
 						log.Warnf("Sensor cannot reconcile due to: %v", err)
 					}
 					s.reconcile.Store(false)
+					log.Infof("Communication with Central stopped with error: %s. Retrying.", err)
 				}
-				log.Infof("Communication with Central stopped with error: %s. Retrying.", err)
+				log.Infof("Communication with Central stopped: %s. Retrying.", err)
```

Comment:
[nit] I think we normally use `%v` for errors

Reply:
I think it's a mess. :-) 
I changed to `%v` but I'm sure other places are using strings.